packages/react/hover-card/src/HoverCard.tsx

```diff
@@ -0,0 +1,289 @@
+
+  const handelMouseLeave = React.useCallback(() => {
+    clearEnterTimer();
+    leaveTimerRef.current = window.setTimeout(() => {
+      if (enterEnvoked.current) {
+        setOpen(false);
+        enterEnvoked.current = false;
+      }
+    }, exitDelayDuration);
+  }, [clearEnterTimer, exitDelayDuration, setOpen]);
+
+  // cleanup any queued state updates on unmount
+  React.useEffect(() => {
+    return () => {
+      clearEnterTimer();
+      clearLeaveTimer();
+    };
+  }, [clearEnterTimer, clearLeaveTimer]);
+
+  return (
+    <PopperPrimitive.Root>
+      <HoverCardProvider
+        contentId={useId()}
+        triggerRef={triggerRef}
+        open={open}
+        onOpenChange={setOpen}
+        onMouseEnter={handleMouseEnter}
+        onMouseLeave={handelMouseLeave}
+        hasCustomAnchor={hasCustomAnchor}
+        onCustomAnchorAdd={React.useCallback(() => setHasCustomAnchor(true), [])}
+        onCustomAnchorRemove={React.useCallback(() => setHasCustomAnchor(false), [])}
+      >
+        {children}
+      </HoverCardProvider>
+    </PopperPrimitive.Root>
+  );
+};
+
+HoverCard.displayName = HOVERCARD_NAME;
+
+/* -------------------------------------------------------------------------------------------------
+ * HoverCardAnchor
+ * -----------------------------------------------------------------------------------------------*/
+
+const ANCHOR_NAME = 'HoverCardAnchor';
+
+type HoverCardAnchorOwnProps = Polymorphic.OwnProps<typeof PopperPrimitive.Anchor>;
+type HoverCardAnchorPrimitive = Polymorphic.ForwardRefComponent<
+  Polymorphic.IntrinsicElement<typeof PopperPrimitive.Anchor>,
+  HoverCardAnchorOwnProps
+>;
+
+const HoverCardAnchor = React.forwardRef((props, forwardedRef) => {
+  const context = useHoverCardContext(ANCHOR_NAME);
+  const { onCustomAnchorAdd, onCustomAnchorRemove } = context;
+
+  React.useEffect(() => {
+    onCustomAnchorAdd();
+    return () => onCustomAnchorRemove();
+  }, [onCustomAnchorAdd, onCustomAnchorRemove]);
+
+  return <PopperPrimitive.Anchor {...props} ref={forwardedRef} />;
+}) as HoverCardAnchorPrimitive;
+
+HoverCardAnchor.displayName = ANCHOR_NAME;
+
+/* -------------------------------------------------------------------------------------------------
+ * HoverCardTrigger
+ * -----------------------------------------------------------------------------------------------*/
+
+const TRIGGER_NAME = 'HoverCardTrigger';
+const TRIGGER_DEFAULT_TAG = 'a';
+
+type HoverCardTriggerOwnProps = Polymorphic.OwnProps<typeof Primitive>;
+type HoverCardTriggerPrimitive = Polymorphic.ForwardRefComponent<
+  typeof TRIGGER_DEFAULT_TAG,
+  HoverCardTriggerOwnProps
+>;
+
+const HoverCardTrigger = React.forwardRef((props, forwardedRef) => {
+  const { as = TRIGGER_DEFAULT_TAG, ...triggerProps } = props;
+  const context = useHoverCardContext(TRIGGER_NAME);
+  const composedTriggerRef = useComposedRefs(forwardedRef, context.triggerRef);
+
+  const trigger = (
+    <Primitive
+      type="a"
```

Comment:
I think that's a leftover from copy pasting `type="button"`?

Reply:
Indeed a pasta error 😅